electron-app/electron-builder.yml

```diff
@@ -28,7 +38,7 @@ linux:
   category: Development
   vendor: Eclipse Foundation, Inc
   target:
-    - deb
+    # - deb
```

Comment:
Should we leave this in as commented out or just remove it?

Reply:
I think we may leave it in actually. Deb does not support the autoupdate however, so we will have to discuss whether we want to keep it without update, remove it, or offer a debian repository.